docs/components/camera/webcam.md

```diff
@@ -133,16 +133,24 @@ Viam supports the following pixel formats:
 
 - I420
 - I444
-- MJPEG
+- MJPEG / MJPG
 - NV12
 - NV21
 - RGBA
-- UYVY
-- YUY2
+- UYVY / Y422
+- YUY2 / YUYV / V422
 - Z16
 
 If your machine is connected to the Viam app, the available pixel formats supported by your camera automatically appear in the **Format** dropdown menu, which is visible when you click the **Show more** button.
 
+You can also manually determine the pixel formats your camera supports by running the following command on the machine you have connected your camera to:
```

Comment:
Just FYI, this will only work on Linux. I'm not sure if that's clarified somewhere.

Reply:
Thanks! It's back on the troubleshooting page, but definitely worth clarifying here 👍